holidays/countries/malaysia.py

```diff
@@ -74,10 +74,15 @@ class Malaysia(
         * [Holidays Act 1951](https://web.archive.org/web/20241202103403/https://www.kabinet.gov.my/bkpp/pdf/akta_warta/1951_12_31_act369.pdf)
         * [Holidays Ordinance (Sabah Cap. 56)](https://web.archive.org/web/20201028045259/https://sagc.sabah.gov.my/sites/default/files/law/HolidaysOrdinance.pdf)
         * [Public Holidays Ordinance (Sarawak Cap. 8)](https://web.archive.org/web/20221208142318/https://www.kabinet.gov.my/bkpp/pdf/akta_warta/sarawak_public_holidays_ord_chapter8.pdf)
-        * [Wikipedia](https://en.wikipedia.org/wiki/Public_holidays_in_Malaysia)
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Malaysia>
         * <https://web.archive.org/web/20250123115300/https://www.nst.com.my/news/nation/2020/03/571660/agongs-birthday-moved-june-6-june-8>
         * <https://web.archive.org/web/20240228225038/https://www.nst.com.my/news/nation/2024/02/1014012/melaka-cm-suggests-declaring-feb-20-federal-public-holiday-mark>
 
+    Checked with:
```

Comment:
Maybe it's too early to update 2027 dates without confirmation from official sources? ([Here](https://web.archive.org/web/20251216120554/https://www.kabinet.gov.my/storage/2025/08/HKA-2026.pdf) is for 2026)

And Birthday of the Governor of Sabah [changed from 1st Sat of Oct to Mar 30 since 2025](https://www.dailyexpress.com.my/news/254584/sabah-public-holiday-update-tyt-s-official-birthday-shifted-to-march-30/).

Reply:
Update: Birthday of the Governor of Sabah has changed before, I'd better make separate PR for it (as I have already collected the sources and data anyway). :)